### DISCUSSION TITLE
Migrating from Sabayon / upgrading

### DISCUSSION THREAD

**Discussion by cream09** [mocaccinoOS/mocaccino#50] [General]
Hi,

I followed the instructions on the mocaccinoOS wiki, but after the "luet upgrade", it fails in the finalizer process with 

`Failed running command: : exec: *sh*: executable file not found in $PATH`

When I try to run luet upgrade again I get the error 

`could not aquire exclusive lock from luet: no such file or directory.` 

The system is after "luet upgrade" unusable.

Thanks in advance

**Comment by joostruis** [comment]
Hello. Thanks for you report. I must say that it has been some time since I tested a migration.
I'll give it another shot soon and see if I can reproduce.

Image for testing that I use is here: https://sourceforge.net/projects/archiveos/files/s/sabayon/Sabayon_Linux_19.03_amd64_Xfce.iso/download

**Comment by joostruis** [comment]
I am able to reproduce this in a VM. Perhaps this problem is caused by the concurrency setting in luet.yaml.

// edit: tried again with concurrency set to 1 (it wasn't set at all,no idea if it made a difference) and ended up with the same broken system. Very hard to catch logs, even though I ran with --debug, my tools like ls and mkdir are all gone.

**Reply by cream09** [reply]
Now everything seems working until the reboot. KDE/plasma doesn't load, I just see my cursor. After "luet oscheck" there are several packages that are missing or incomplete. How can I solve that?

Thanks

**Comment by cream09** [comment]
Thanks for your answer. How can I update my system then? I'm on Kernel 5.6 with KDE.
